api/v1alpha1/oauth2client_json.go

```diff
@@ -0,0 +1,11 @@
+package v1alpha1
+
+// OAuth2ClientJSON represents an OAuth2 client digestible by ORY Hydra
+type OAuth2ClientJSON struct {
```

Comment:
This type does not really belong here. It's not tied to the v1alpha1 version at all. I would move it to top-level "hydra" package

Reply:
Sure thing, will be moved.